src/shared/setup.js

```diff
@@ -58,9 +58,9 @@ const initialize = async function ({service, port, modules = [], runMigrations =
   await connectDatabase()
   await Sidecar.connect(service)
   const server = await createServer(port, modules)
-  if (service === 'api') {
+  /* if (service === 'api') {
     await Account.createLedgerAccount(Config.LEDGER_ACCOUNT_NAME, Config.LEDGER_ACCOUNT_PASSWORD, Config.LEDGER_ACCOUNT_EMAIL)
```

Comment:
Now that this is disabled, are we not going to create a default account/participant?

Reply:
Yes, but it was disabled only to pass the 149 story without throwing errors. I will take care of it Monday

Reply:
Yes, but it was disabled only to pass the 149 story without throwing errors. I will take care of it Monday